lib/joke.rb

```
@@ -1,3 +1,42 @@
+require 'pry'
 class Joke
 
+  def initialize(jokes = {})
```

Don't set a default value if you don't need to. Would rather see this error out than fail silently. This default variable means to someone using your class that you're saying to them "Hey! If you don't want to pass me any arguments, that's fine with me!" It is not! It is not fine with us! Pass us arguments.